docs/reference-guides/block-api/block-metadata.md

```diff
@@ -601,6 +601,16 @@ PHP file to use when rendering the block type on the server to show on the front
 -   `$content` (`string`): The block default content.
 -   `$block` (`WP_Block`): The block instance.
 
+An example implementation of the `render.php` file defined with `render` could look like:
+
+```php
+<div <?php echo get_block_wrapper_attributes(); ?>>
+	<?php echo esc_html( $attributes['label'] ); ?>
+</div>
+```
+
+_Note: This file loads for every instance of the block type when rendering a single page. Accounting for that is essential when declaring functions or classes in the file. The simplest way to avoid the risk of errors is to consume that shared logic from another file._
```

Comment:
I think this here should be reworded a little. In its current form it's confusing because a dynamic block could also be rendered as part of a template etc. 

It really is about the fact that the PHP in that file only gets included when the block is rendered somewhere.

Reply:
Yes, I intended to explain that the dynamic block might need to be rendered several times, so the risk of a function/class re-declaration needs to be taken into account.

Reply:
How about [a19b7c4](https://github.com/WordPress/gutenberg/pull/53973/commits/a19b7c47f4a99e5af0ab11973606bfe7df996919):

> This file loads for every instance of the block type when rendering the page HTML on the server.

Reply:
I like it :)